pkgs/development/libraries/libbgcode/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, heatshrink
+, zlib
+, boost
+, catch2
+}:
+
+let
+  src = fetchFromGitHub {
```

Comment:
since this var is not reused you can put it after version

Reply:
heatshrink leftovers, fixed